pandas/io/parsers.py

```diff
@@ -1387,6 +1390,20 @@ def _wrap_compressed(f, compression, encoding=None):
             data = bz2.decompress(f.read())
             f = StringIO(data)
         return f
+    elif compression == 'zip':
```

Comment:
this seems quite duplicative (I know it was there before). any way you can simply use ``_get_handle``? (which already has this embeded)